### DISCUSSION TITLE
install script doesn't run

### DISCUSSION THREAD

**Discussion by caiotizio** [DefGuard/defguard#682] [General]
hi there,
i'm trying to install defguard server on debian 12 container in proxmox.
i followed these [docs](https://defguard.gitbook.io/defguard/tutorials/step-by-step-setting-up-a-vpn-server)  but when i launch `curl --proto '=https' --tlsv1.2 -sSf -L https://raw.githubusercontent.com/DefGuard/deployment/main/docker-compose/setup.sh -O && bash setup.sh` nothing happens, just a clear of the shell. i found some  empty logs in the current directory. 
i tried the same procedure on arm64 and amd64 architectures with same results.
what is going wrong?

**Comment by teon** [comment]
It will not run in a container. If want isolation please use a VM in proxmox that has the official Docker installation (and not distribution packages).
